client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionEventForm/ContactForm/ContactForm.tsx

```diff
@@ -27,13 +29,15 @@ const ContactForm: React.FC<Props> = (props: Props): JSX.Element => {
     const classes = useStyles();
     const formClasses = useFormStyles();
 
-    const {interactionEventDialogData, setInteractionEventDialogData} = useContext(InteractionEventDialogContext);
-    const {contacts} = interactionEventDialogData;
+    const contactTypes = useSelector<StoreStateType, Map<number, ContactType>>(state => state.contactTypes);
+
+    const { interactionEventDialogData, setInteractionEventDialogData } = useContext(InteractionEventDialogContext);
+    const { contacts } = interactionEventDialogData;
     const contact = contacts[updatedContactIndex];
     const {extraInfo, contactType, lastName, firstName, phoneNumber, id} = contact;
 
     React.useEffect(() => {
-        onChange(ContactType.TIGHT, InteractionEventContactFields.CONTACT_TYPE);
+        contactType === -1 && onChange(Array.from(contactTypes.keys())[0], InteractionEventContactFields.CONTACT_TYPE);
```

Comment:
maybe take the invalid type from const or -
```suggestion
        contactType === defaultContact.contactType && onChange(Array.from(contactTypes.keys())[0], InteractionEventContactFields.CONTACT_TYPE);
```